src/cclib/parser/data.py

```diff
@@ -35,6 +35,9 @@ class ccData(object):
         entropy -- entropy (float, hartree/particle)
         etenergies -- energies of electronic transitions (array[1], 1/cm)
         etoscs -- oscillator strengths of electronic transitions (array[1])
+        eteltrdips -- electric transition dipoles of electronic transitions (array[2], ebohr)

```

Comment:
I would just call these `etdips`, `etveldips` and `etmagdips`. Any thoughts (@berquist or others)?

Reply:
Sounds good!

Reply:
Did this in #803